dev-docs/source/Testing/ElementalAnalysis/ElementalAnalysisTests.rst

```diff
@@ -0,0 +1,26 @@
+.. _elemental_analysis_gui_testing:
```

Comment:
Nice! A good thing we can build on :+1: 

I wonder if we should update the PR template as a reminder to reviewers to check whether these types of document should be updated.

Reply:
probably worth doing along with a reminder for QTest (if possible)